STMicroelectronics/STM32L4_L4+/app/az_ulib_dm/inc/dm_1_model.h

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license.
+// See LICENSE file in the project root for full license information.
+
+/********************************************************************
+ * This code was auto-generated from dm v1 DL and shall not be
+ * modified.
+ ********************************************************************/
+
+#ifndef DM_1_MODEL_H
+#define DM_1_MODEL_H
+
+#include "az_ulib_ipc_api.h"
+#include "az_ulib_result.h"
+#include "azure/az_core.h"
+
+#ifdef __cplusplus
+#include <cstdint>
+extern "C"
+{
+#else
+#include <stdint.h>
+#endif
+
+/*
+ * interface definition
+ */
+#define DM_1_INTERFACE_NAME "dm"
+#define DM_1_INTERFACE_VERSION 1
+#define DM_1_CAPABILITY_SIZE 2
+
+/*
+ * Define install command on dm interface.
+ */
+#define DM_1_INSTALL_COMMAND (az_ulib_capability_index)0
+#define DM_1_INSTALL_COMMAND_NAME "install"
+#define DM_1_INSTALL_PACKAGE_NAME_NAME "package_name"
```

Comment:
why PACKAGE_NAME_NAME?

Reply:
This is the name of the capability "package_name".